src/main.js

```diff
@@ -145,6 +147,35 @@ class PercyClient {
     return this._httpPost(`${this.apiUrl}/builds/${buildId}/resources/`, data);
   }
 
+  uploadResources(buildId, contents) {
+    const _this = this;
+    function* generatePromises() {
+      for (const content of contents) {
+        yield _this.uploadResource(buildId, content);
+      }
+    }
+
+    const concurrency = 2;
```

Comment:
Throw this in a module level constant.

Also, I think we should probably accept an `options` argument and then do something like:

```js
options = options || {};
let concurrency = options.concurrency || DEFAULT_CONCURRENCY;
```